packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -328,8 +375,7 @@ describeCompat("GC data store sweep tests", "NoCompat", (getTestObjectProvider)
 					summarizingContainer,
 					summarizer,
 					summaryVersion: unreferencedSummaryVersion,
-				} = await summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs);
-				await sendOpToUpdateSummaryTimestampToNow(summarizer);
+				} = await summarizationWithUnreferencedDataStoreAfterTime();
 				const sendingContainer = await loadContainer(unreferencedSummaryVersion);
```

Comment:
I'm not thinking too critically about removing this "send op" call, assuming the tests provide coverage for themselves, essentially.

Reply:
It was not required. The summarizationWithUnreferencedDataStoreAfterTime does that already before returning. Calling it again was just redundant.